docs/reference/developer_guide/developing_custom_catalogs.md

```diff
@@ -25,6 +25,9 @@ There are two ways of adding a custom catalog to Shift:
 - **As a User Catalog**: Use the [Catalog Manager](../catalogs#the-catalog-manager) to source the catalog file and add it to Shift. This catalog is saved in the user preferences.
 - **As an Environment Catalog**: Add the **path to the directory** containing the catalog file to the `SHIFT_CATALOG_PATH` environment variable. All files in this path containing a `catalog` object will be identified as custom catalogs and loaded by Shift.
 
+>[!WARNING]
+> Avoid naming custom catalogs after existing Python modules, as this may cause conflicts between modules in Shift. It is recommended to add a prefix or suffix to custom catalog names for easier identification.
+
```

Comment:
```suggestion
>[!WARNING]
> It is strongly suggested to avoid naming custom catalogs after existing Python modules or packages, as this may cause namespace clashes. It is recommended to add a prefix or suffix to custom catalog names for unique identification of the catalog module.

```